Assets/BossRoom/Scripts/Client/Net/ClientGameNetPortal.cs

```diff
@@ -211,6 +218,91 @@ public static bool StartClientRelayMode(GameNetPortal portal, string roomKey, ou
             return true;
         }
 
+        public static bool StartClientUnityRelayMode(GameNetPortal portal, string joinCode, out string failMessage)
+        {
+
+            var chosenTransport  = NetworkManager.Singleton.gameObject.GetComponent<TransportPicker>().UnityRelayTransport;
+            NetworkManager.Singleton.NetworkConfig.NetworkTransport = chosenTransport;
+
+            switch (chosenTransport)
+            {
+                case UnityTransport utp:
+                    Debug.Log($"Setting Unity Relay client with join code {joinCode}");
+                    Task t = Task.Run( () => {UnityServices.InitializeAsync();});

```

Comment:
"Initialize" seems like a one time thing. we initialize each time we connect? What if we finish a game and reconnect to another host, in the same run?

Reply:
This function is not used at all